<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="HS18_CRM login test suite" duration-ms="10187" started-at="2016-04-25T05:04:32Z" finished-at="2016-04-25T05:04:42Z">
    <groups>
    </groups>
    <test name="Test For Login to CRM" duration-ms="10187" started-at="2016-04-25T05:04:32Z" finished-at="2016-04-25T05:04:42Z">
      <class name="com.hs18.extentreports.tests.LoginTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.hs18.extentreports.tests.LoginTest@66d33a]" name="setUp" is-config="true" duration-ms="7605" started-at="2016-04-25T10:34:25Z" finished-at="2016-04-25T10:34:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="failingTest(java.lang.String, java.lang.String)[pri:0, instance:com.hs18.extentreports.tests.LoginTest@66d33a]" name="failingTest" duration-ms="5328" started-at="2016-04-25T10:34:32Z" description="Performs an unsuccessful login and checks the resulting error message (fails)" finished-at="2016-04-25T10:34:38Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[wrong]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[credentials]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [This is not the error message you're looking for.] but found [Value is required.Enter Valid value.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [This is not the error message you're looking for.] but found [Value is required.Enter Valid value.]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertEquals(Assert.java:125)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at org.testng.Assert.assertEquals(Assert.java:188)
	at com.hs18.extentreports.tests.LoginTest.failingTest(LoginTest.java:43)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:112)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:176)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- failingTest -->
        <test-method status="PASS" signature="testFailingLogin(java.lang.String, java.lang.String)[pri:0, instance:com.hs18.extentreports.tests.LoginTest@66d33a]" name="testFailingLogin" duration-ms="2406" started-at="2016-04-25T10:34:38Z" description="Performs an unsuccessful login and checks the resulting error message (passes)" finished-at="2016-04-25T10:34:40Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[wrong]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[credentials]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testFailingLogin -->
        <test-method status="PASS" signature="testSuccessfulLogin(java.lang.String, java.lang.String)[pri:0, instance:com.hs18.extentreports.tests.LoginTest@66d33a]" name="testSuccessfulLogin" duration-ms="2419" started-at="2016-04-25T10:34:40Z" description="Performs a successful login and checks Home page is opened" finished-at="2016-04-25T10:34:42Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[manish3]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[hsnmanish]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSuccessfulLogin -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.hs18.extentreports.tests.LoginTest@66d33a]" name="tearDown" is-config="true" duration-ms="570" started-at="2016-04-25T10:34:42Z" finished-at="2016-04-25T10:34:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.hs18.extentreports.tests.LoginTest -->
    </test> <!-- Test For Login to CRM -->
  </suite> <!-- HS18_CRM login test suite -->
</testng-results>
